cudamapper/src/cudamapper_utils.cpp

```diff
@@ -38,7 +38,7 @@ std::vector<gw_string_view_t> split_into_kmers(const gw_string_view_t& s, const
     const std::size_t kmer_count = s.length() - kmer_size + 1;
     std::vector<gw_string_view_t> kmers;
 
-    if (s.length() < kmer_size)
+    if (s.length() < static_cast<size_t>(kmer_size))
```

Comment:
We actually have a functions which gets signed size of an array: https://github.com/clara-parabricks/GenomeWorks/blob/df8b1f38e48e2aacc13967cfa863f036c1dd37b4/common/base/include/claraparabricks/genomeworks/utils/signed_integer_utils.hpp#L31